test/JDBC/expected/BABEL-3222.out

```diff
@@ -0,0 +1,69 @@
+create table test_tsql_const(
+	c_bit sys.bit check(c_bit <> cast(1 as sys.bit)),
+	c_int int check(c_int < 10),
+);
+go
+
+create table test_tsql_cast(
+	c_float float check(c_float < cast(133.230182309832423 as float) and c_float < cast(133.230182309832423 as float(7))),
+	c_real real check(c_real < cast(133.230182309832423 as real)),
+	c_decimal decimal check(c_decimal < cast(133.230182309832423 as decimal) and c_decimal < cast(133.230182309832423 as decimal(5,3))),
+	c_numeric numeric check(c_numeric < cast(133.230182309832423 as numeric) and c_numeric < cast(133.230182309832423 as numeric(5,3))),
+);
+go
+
+create table test_tsql_collate(
+	c_varchar varchar check(c_varchar <> cast('sflkjasdlkfjf' as varchar(12)) COLLATE bbf_unicode_cp1_ci_as),
+	c_char char check(c_char <> cast('sflkjasdlkfjf' as char(7)) COLLATE bbf_unicode_cp1_ci_as),
+	c_nchar nchar check(cast(c_nchar as nchar(7)) <> cast('sflkjasdlkfjf' as nchar(7)) COLLATE bbf_unicode_cp1_ci_as),
+);
+go
+
+create table test_null(a int, check(a IS NOT NULL));
+go
+
+select tsql_get_constraintdef(oid) from pg_constraint where contype='c' and conrelid = (select oid from pg_class where relname='test_null');
+go
+~~START~~
+text
+((a IS NOT NULL))
+~~END~~
+
+
+select tsql_get_constraintdef(oid) from pg_constraint where contype='c' and conrelid = (select oid from pg_class where relname='test_tsql_const');
+go
+~~START~~
+text
+((c_bit <> CAST((1) AS "bit")))
+((c_int < 10))
+~~END~~
+
+
+select tsql_get_constraintdef(oid) from pg_constraint where contype='c' and conrelid = (select oid from pg_class where relname='test_tsql_collate');
+go
+~~START~~
+text
+((c_varchar <> (CAST( AS "varchar"(12)) COLLATE bbf_unicode_cp1_ci_as)))
```

Comment:
I see no value being returned for string types. for example  `cast('sflkjasdlkfjf' as varchar(12)) `is converted to `CAST( AS "varchar"(12)) `

Moreover does the double quotes comply with SQL Server?

Reply:
This was missed in Harsh's last commit, it has been fixed now. Double quotes are removed as it doesn't comply with SQL Server.

Reply:
Fixed now.